### PR TITLE
AAP-5155

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Affects the "Configuring proxy support for Ansible Automation Platform" chapter of the installation guide (`/titles/aap-installation-guide/`)

- Eliminate the nested assembly (`assembly-configure-known-proxies.adoc`) and include the modules directly from `master.adoc``.
- Move the websockets content out of `assembly-configuring-proxy-support.adoc` and include it in `master.adoc`
- Replace references to "Tower" with "automation controller"

### CODE REVIEW

File: downstream/modules/platform/con-known-proxies.adoc [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 
-= About known proxies
+= Known proxies
 
 
 [role="_abstract"]
 
-When Tower is configured with `REMOTE_HOST_HEADERS = ['HTTP_X_FORWARDED_FOR', 'REMOTE_ADDR', 'REMOTE_HOST']`, it assumes that the value of `X-Forwarded-For` has originated from the proxy/load balancer sitting in front of Tower. In a scenario where Tower is still reachable without use of the proxy/load balancer or when the proxy does not validate the header, `X-Forwarded-For` can be spoofed fairly easily to fake the originating IP addresses. Using `HTTP_X_FORWARDED_FOR` in the `REMOTE_HOST_HEADERS` setting poses a vulnerability that essentially gives users access to certain resources that they should not have.
+When {ControllerName} is configured with `REMOTE_HOST_HEADERS = ['HTTP_X_FORWARDED_FOR', 'REMOTE_ADDR', 'REMOTE_HOST']`, it assumes that the value of `X-Forwarded-For` has originated from the proxy/load balancer sitting in front of {ControllerName}.
+In a scenario where {ControllerName} is still reachable without use of the proxy/load balancer or when the proxy does not validate the header, `X-Forwarded-For` can be spoofed fairly easily to fake the originating IP addresses.
```
Comment (ianf77) [inline downstream/modules/platform/con-known-proxies.adoc:10]: I was never sure about the word 'spoofed' it doesn't sound particularly technical!

Comment (ariordan-redhat) [reply downstream/modules/platform/con-known-proxies.adoc:10]: You're right: that whole sentence could do with a re-write.

Comment (ariordan-redhat) [reply downstream/modules/platform/con-known-proxies.adoc]: @ianf77 
Could you please review the new change I made?

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**